.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,46 @@
-<!-- Thank you for your contribution! Please review https://github.com/ray-project/ray/blob/master/CONTRIBUTING.rst before opening a pull request. -->
+<!-- Thank you for contributing to Ray! 🚀 -->
+<!-- Please review https://github.com/ray-project/ray/blob/master/CONTRIBUTING.rst before opening a pull request. -->
+<!-- 💡 Tip: Mark as draft if you want early feedback, or ready for review when it's complete -->
 
-<!-- Please add a reviewer to the assignee section when you create a PR. If you don't have the access to it, we will shortly find a reviewer and assign them to your PR. -->
+## Description
 
-## Why are these changes needed?
+<!-- Briefly describe what this PR accomplishes and why it's needed -->
 
-<!-- Please give a short summary of the change and the problem this solves. -->
+## Related issues
 
-## Related issue number
+<!-- Link related issues: "Fixes #1234", "Closes #1234", or "Related to #1234" -->
 
-<!-- For example: "Closes #1234" -->
+## Type of change
 
-## Checks
+- [ ] Bug fix 🐛
+- [ ] New feature ✨
+- [ ] Enhancement 🚀
+- [ ] Code refactoring 🔧
+- [ ] Documentation update 📖
 
-- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
-- [ ] I've run pre-commit jobs to lint the changes in this PR. ([pre-commit setup](https://docs.ray.io/en/latest/ray-contribute/getting-involved.html#lint-and-formatting))
-- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
-    - [ ] I've added any new APIs to the API Reference. For example, if I added a
-           method in Tune, I've added it in `doc/source/tune/api/` under the
-           corresponding `.rst` file.
-- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
-- Testing Strategy
-   - [ ] Unit tests
-   - [ ] Release tests
-   - [ ] This PR is not tested :(
+## Breaking changes
```

Comment:
Maybe we can consolidate this section in the "description" section for brevity purposes?

Reply:
I moved it to the checklist so it's not so jarring as part of the description.